cli/cmd/vuln_container_list_assessments.go

```diff
@@ -385,6 +390,18 @@ func (v *treeCtrVuln) ParseData(data []api.VulnerabilityContainer) {
 	}
 }
 
+// Replace updates an existing ctrVuln in the treeCtrVuln slice with a new ctrVuln
+func (v *treeCtrVuln) Replace(old ctrVuln, new ctrVuln) {
+	current := *v
+	for i, ctrVuln := range current {
+		if ctrVuln.EvalGUID == old.EvalGUID {
+			current[i] = new
+			break
+		}
+	}
+	*v = current
+}
```

Comment:
This is what I think you could do instead:
```suggestion
func (v treeCtrVuln) Replace(old ctrVuln, new ctrVuln) {
	for i, ctrVuln := range v {
		if ctrVuln.EvalGUID == old.EvalGUID {
			v[i] = new
			break
		}
	}
}
```